### DISCUSSION TITLE
List roles yang dibutuhkan untuk leconfe

### DISCUSSION THREAD

**Discussion by gokilalma** [OpenSynergic/leconfe#54] [General]
1. Admin
2. Conference Manager (in OJS as Journal Manager)
3. Director (in OJS as Journal Editor)
4. Track director (in OJS as Section Editor)
5. Reviewer
6. Author

**Comment by rahmanramsi** [comment]
#4

**Comment by rahmanramsi** [comment]
@gokilalma tolong detailkan tugasnya masing" role

**Reply by gokilalma** [reply]
**Admin**
Bertanggung jawab dalam sifat yang sangat teknis, memasang plugin, membuat/menghapus conference, setup email dll. 
Sebagai masukan, kalo bisa admin tidak perlu terlalu bisa mengakses ke roles lain lain, seperti mengelola submission. Namun tetap dapat mengelola hak roles user.


**Conference Manager (in OJS as Journal Manager)**
Mengatur dan mengelola informasi jurnal, seperti judul, deskripsi, dan logo, Menentukan kebijakan dan prosedur yang berlaku dalam jurnal, Mengelola aspek-aspek teknis, seperti konfigurasi tema, email, user.


**Director (in OJS as Journal Editor)**
Fokus dalam mengatur dan menghandle setiap submission yang masuk, editor dapat menerima, menolak, dan menghapus submission.
Editor juga dapat menambahkan, Track director atau reviewer di setiap proses submission.

Namun pada roles director dapat mengakses dan mengatur terkait konfigurasi informasi aplikasi conference.


**Track director (in OJS as Section Editor)**
Sama dengan directorm hanya saja user ini dapat digunakan setelah menerima assign dari seorang Conference manager atau director.


**Reviewer**
Hanya bertugas memberikan review terhadap penugasan yang diberikan. memberikan hasil penilaian dan memberikan rekomendasi dari kesimpulan.


**Author**
-
